### DISCUSSION TITLE
Should we enable ssc? (server side characters)

### DISCUSSION THREAD

**Discussion by ZsoltiHUB** [Pryaxis/TShock#2918] [Q&A] [answered]
Hi, me and my friend were wondering if we should enable ssc or not. I'm asking your experiences with ssc. We are doing a vanilla-like server but we dont want our players to cheat in items. I've seen in the docs that data corruption might happen, but what are the chances?

**Comment by sgkoishi** [comment, marked as answer]
Not quite likely but it exists, backup is the best solution. I think #2894 fix it but not so sure.
SSC prevents the use of local data, but the players are still able to 'cheat' using third-party tools.
The server can only manipulate the inventory and many other stats when SSC is enabled.

**Reply by sgkoishi** [reply]
Also note that data loss might happen if administrative tools are used incorrectly (most commonly "playing as superadmin" is a 100% data loss)
